optimade/validator/__init__.py

```diff
@@ -98,6 +99,12 @@ def validate():  # pragma: no cover
         help="Additional HTTP headers to use for each request, specified as a JSON object.",
     )
 
+    parser.add_argument(
+        "--version",
+        action="version",
+        version=f"optimade-validator running from optimade-python-tools v{__version__}, OPTIMADE specification v{__api_version__}.",
```

Comment:
```suggestion
        version=f"optimade-validator running from optimade-python-tools v{__version__}, applying OPTIMADE specification v{__api_version__}.",
```

Or `using`?

Reply:
Maybe also lose the comma then? I don't know. You're the native 😅

Reply:
I added "which implements" as connecting words